.eslintrc

```diff
@@ -32,7 +30,6 @@
         "func-names": [0],
         "func-style": [0, "declaration"],
         "generator-star-spacing": [2, "after"],
-        "global-strict": [2, "always"],
```

Comment:
1. Depracated: http://eslint.org/docs/rules/global-strict.html
2. Does not work well with ES6 modules without `use strict`